models/Trip.js

```diff
@@ -1,4 +1,4 @@
-const { boolean, required } = require('joi');
+// const { boolean, required } = require('joi');
```

Comment:
No change needed but as a reminder its usually better to remove commented code instead of leaving it around. Remember, everything is version controlled.